src/controllers/api/account.ts

```diff
@@ -189,7 +189,9 @@ export class AccountController {
 			// Even if customer was created before for such user but the process was interruted somehow - we need to create it again
 			// Cause we don't know the state of the customer in this case
 			// 2.1.1. Create customer
-			customer = (await CustomerService.instance.create(logToUserEmail)) as CustomerEntity;
+			// I’m setting the "name" field to an empty string on the current CustomerEntity because it is non-nullable.
+			//  we will populate the customer's "name" field using the response from the Stripe account creation in account-submitter.ts.
+			customer = (await CustomerService.instance.create('', logToUserEmail)) as CustomerEntity;
```

Comment:
This `name` field should be the organisation/customer name that they set in settings. If you need a default to initialise (say, if the customer hasn't yet gone to that page and set it), could you instead set it to the `name` property of the Logto user? This might sometimes have a name, sometimes be blank. Better than just straight up defaulting to blank.

Reply:
Yeah, make sense. 👍🏾

Reply:
But, you it will get override by this bit in `account-submitter.ts` which uses name from stripe. But, it does make sense to keep the name from "stripe" as it is mandatory in checkout and "customer" table indeed referring to the stripe customer.
```
await CustomerService.instance.update(data.customerId, data.name, undefined, undefined, account.id);
```